src/app/custom/translations.tsv

```diff
@@ -1050,7 +1061,11 @@
 "pc_post-v1-tag-cloud-en_title"	"English terms extraction"	"Extraction de termes Anglais"
 "pc_post-v1-tag-cloud-en_description"	"Extracts the most frequent English terms from a corpus of documents "	"Propose une extraction des termes anglais les plus fréquents à l'échelle d'un corpus de documents"
 "pc_post-v1-tag-cloud-fr_title"	"French terms extraction"	"Extraction de termes Français"
-"pc_post-v1-tag-cloud-fr_description"	"Extracts the most frequent French terms from a corpus of documents "	"Propose une extraction des termes anglais les plus fréquents à l'échelle d'un corpus de documents "
+"pc_post-v1-tag-cloud-fr_description"	"Extracts the most frequent French terms from a corpus of documents "	"Propose une extraction des termes français les plus fréquents à l'échelle d'un corpus de documents"
+"pc_post-v1-textClustering _title"  "Extracting clusters from a dataset"  "Extraction de clusters d’un corpus"
+"pc_post-v1-textClustering_description"  "Extracts several groups (clusters) from a corpus in order to classify the different texts according to their similarity. A document is present in only one group"  "Extrait plusieurs groupes (clusters) d’un corpus afin d’y classer les différents textes en fonction de leur similarité. Un document est présent dans un seul groupe"
+"pc_post-v1-ldaClass_title"  "Extracting themes from a dataset"  "Extraction de thématiques d’un corpus"
+"pc_post-v1-ldaClass_description"  "Extracts themes from a corpus: a theme (or topic) is characterized by ten words. Once the themes have been extracted, each document is assigned one or more themes."  "Extrait des thématiques d’un corpus : une thématique (ou topic) est caractérisée par dix mots. Une fois les thématiques extraites, chaque document se voit attribuer une ou plusieurs thématique(s)."
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Tabulation instead of 2 spaces
```suggestion
"pc_post-v1-tag-cloud-fr_description"	"Extracts the most frequent French terms from a corpus of documents"	"Propose une extraction des termes français les plus fréquents à l'échelle d'un corpus de documents"
"pc_post-v1-textClustering _title"	"Extracting clusters from a dataset"	"Extraction de clusters d’un corpus"
"pc_post-v1-textClustering_description"	"Extracts several groups (clusters) from a corpus in order to classify the different texts according to their similarity. A document is present in only one group"	"Extrait plusieurs groupes (clusters) d’un corpus afin d’y classer les différents textes en fonction de leur similarité. Un document est présent dans un seul groupe"
"pc_post-v1-ldaClass_title"	"Extracting themes from a dataset"	"Extraction de thématiques d’un corpus"
"pc_post-v1-ldaClass_description"	"Extracts themes from a corpus: a theme (or topic) is characterized by ten words. Once the themes have been extracted, each document is assigned one or more themes."	"Extrait des thématiques d’un corpus : une thématique (ou topic) est caractérisée par dix mots. Une fois les thématiques extraites, chaque document se voit attribuer une ou plusieurs thématique(s)."
```

Reply:
Spaces are still there:
![image](https://github.com/user-attachments/assets/3b003f90-dfde-4093-b3a3-d60a88e38beb)

I think you clicked on `Resolve conversation` instead of `Commit suggestion`.